### ISSUE TITLE
Exec format error

### ISSUE THREAD

**Issue by Ploppz** [mozart/mozart2#322]
```
$ ozc -v
Mozart Compiler 2.0.1+build.8.db41d83e-dirty (Mon, 22 Oct 2018 15:20:43 +0200) playing Oz 3
```
(mozart2 installed from here https://aur.archlinux.org/packages/mozart2-git/)

OS: Arch Linux with kernel version 5.0.2
CPU: 64-bit (AMD Ryzen 5 1600)

My problem is that when I run `ozc -x Game.oz`, I get an executable file `Game` that I cannot execute - when I try to execute it I get the error `bash: ./Game: cannot execute binary file: Exec format error`. `file Game` says it's just "data". 

Minimal `Game.oz`:
```
functor
import
   System
   Application
define
   {System.showInfo 'hi'}
   {Application.exit 0}
end

```

**Comment by layus** [comment]
Yes, the -x flag is indeed not implemented correctly and has currently the same effect as -c except that it produces a file named Game instead of Game.ozf produced by -c. But as for now, both files contain exactly the same content. You can check that with `ozengine Game`.

**Comment by layus** [comment]
See https://github.com/mozart/mozart2/blob/0b3046411ad7b070c08824f91425e304f19d8648/lib/main/sys/Pickle.oz#L66-L71

**Comment by Ploppz** [comment]
Thanks! So I need to use `ozengine` to run the file.

**Comment by layus** [comment]
Yes, that's the way it is for now.
You can leave this bug open until we fix it ;-)
